acousticbrainz/views/api.py

```diff
@@ -7,11 +7,32 @@
 api_bp = Blueprint('api', __name__)
 
 
+@api_bp.route("/<uuid:mbid>/meta", methods=["GET"])
```

Comment:
yes, there was a possibility that we could add other metadata here. any ideas for other metadata? :)